### DISCUSSION TITLE
Can I use chainWebpack in Vue Cli5?

### DISCUSSION THREAD

**Discussion by kingyue737** [vuejs/vue-cli#6910] [Q&A] [answered]
It seems that webpack-chain does not support webpack5 https://github.com/neutrinojs/webpack-chain/issues/266. Can I use chainWebpack in `vue.config.js` in vue cli5 project?

**Comment by Viggo95** [comment, marked as answer]
If you are using chainWebpack to configure the key that both webpack4 and webpack5 have, there is no problem

But adding the new key added by webpakc5 does will  fail
